runtime-light/stdlib/job-worker/job-worker-api.cpp

```diff
@@ -68,7 +68,7 @@ task_t<int64_t> kphp_job_worker_start_impl(string request, double timeout, bool
     const string response{(co_await wait_with_timeout_t{task_t<string>::awaiter_t{std::addressof(fetch_task)}, timeout}).value_or(string{})};
 
     tl::TLBuffer tlb{};
-    tlb.store_bytes(response.c_str(), static_cast<size_t>(response.size()));
+    tlb.store_bytes({response.c_str(), static_cast<size_t>(response.size())});
```

Comment:
Maybe should add new method for string that returns size that is casted to `size_t`?

Reply:
We can't overload a function by its return type

Reply:
We can call it in another way
`size_cast<T>`, `std_size` or smth like that

Reply:
I'd rather change the signature of `string::size()` method so that it returns `size_t`. But it requires a lot of refactoring in the current runtime

Reply:
Yeah, that's why new method has only benefits
Just to think about, not really related to PR